starter-examples/starter-ranking.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
<p>Why are you using f-strings for names here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/truera-examples/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
Accidentally included this while debugging. Removed!